.circleci/config.yml

```diff
@@ -141,8 +140,11 @@ jobs:
       - run:
           name: Run PHPUnit tests
           command: |-
-            mkdir -p build/logs/tmp build/cov
-            phpdbg -qrr vendor/bin/phpunit --coverage-php build/cov/coverage.cov --log-junit build/logs/tmp/coverage.xml --colors=always
+            mkdir -p build/logs/parallel build/logs/tmp build/cov
+            split_tests=$(find tests -name '*Test.php' -not -path 'tests/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtensionTest.php' | circleci tests split --split-by=timings)
+            export PARALLEL='-j10% --joblog build/logs/parallel/jobs.log --rpl {_}\ s/\\//_/g;'
+            phpunit_cmd='phpdbg -qrr vendor/bin/phpunit --coverage-php build/cov/coverage-{_}.cov --log-junit build/logs/tmp/{_}.xml --colors=always {}'
+            echo "$split_tests" | parallel "$phpunit_cmd" || echo "$split_tests" | parallel --retry-failed "$phpunit_cmd"
```

Comment:
`parallel --citation`?

Reply:
https://www.gnu.org/software/parallel/man.html#citation

We don't pay 10000 EUR. 🙈